src/main/features/response/routes/decide-how-you-will-pay.ts

```diff
@@ -68,6 +68,9 @@ export default express.Router()
 
           const { externalId } = req.params
           switch (form.model.option) {
+            case DefendantPaymentType.IMMEDIATELY:
+              res.redirect(Paths.taskListPage.evaluateUri({ externalId: externalId }))
+              break
```

Comment:
I'd rather use `return` here. 

Why?
If you add anything after this switch-case it may be executed before you are redirected.

Reply:
you can do return res.redirect for a one-liner